subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -842,7 +842,14 @@ u8_t ll_adv_enable(u8_t enable)
 				   ticks_anchor, 0,
 				   HAL_TICKER_US_TO_TICKS((u64_t)interval *
 							  625),
-				   TICKER_NULL_REMAINDER, TICKER_NULL_LAZY,
+				   TICKER_NULL_REMAINDER,
+#if !defined(CONFIG_BT_TICKER_COMPATIBILITY_MODE) && \
+	!defined(CONFIG_SOC_COMPATIBLE_NRF)
```

Comment:
I don't think this should depend on SOC_COMPATIBLE_NRF. What would break if that guard was removed?

Reply:
> I don't think this should depend on SOC_COMPATIBLE_NRF. What would break if that guard was removed?

I guess it would be OK to not have the !SOC_COMPATIBLE_NRF as dependency - and it makes good sense, as long as other platforms can accept the additional ticker updates when advertiser does not get air-time. @cvinayak, what is your opinion?

Reply:
its ok to not have !SOC_COMPATIBLE_NRF. Instead use CONFIG_BT_CTLR_LOW_LAT which covers nRF51, and will avoid this code block for nRF51